DigitalEdge.Repository/Interface/IAccountRepository.cs

```diff
@@ -24,7 +24,7 @@ public interface IAccountRepository
         string CreateFacility(Facility addfacility);
         string servicecreateuser(ServicePoint addfacilityuser);
         void updateUser(Users adduser);
-        void AddAttendanceService(Appointment adduser);
+        void UpdateAppointment(Appointment adduser);
```

Comment:
Kindly, revert to the previous setup. Method names were updated to reflect separate functionality according to implemented features, i.e. AddAppointment, EditAppointment, and AddAttendance.